README.md

```diff
@@ -66,7 +66,9 @@ ROM CRC32: 1394F57E
 
 A link to the BPS can be found on the [releases page](https://github.com/kirjavascript/TetrisGYM/releases).
 
-The BPS produces a file with an MMC1 header, but it also works when treated as CNROM.
+The BPS produces a file with its header specifying MMC1 with fixed PRG (mapper 1:5), but it also works when specified as CNROM with bus conflicts (mapper 3:2).
```

Comment:
I don't think we need to add additional information about the mappers. Most people dont care about what the mapper is and the people that do will already know this stuff.

Reply:
Understood. Will revert that change.